lib/src/main/kotlin/com/swmansion/starknet/crypto/HashMethod.kt

```diff
@@ -0,0 +1,18 @@
+package com.swmansion.starknet.crypto
+
+import com.swmansion.starknet.data.types.Felt
+
+enum class HashMethod {
```

Comment:
I'm not sure whether `HashMethod` or `HashFunction` is a more suitable name for this enum

Reply:
Maybe `HashType` instead? method/function kind of indicates we are dealking with a function, and not an enum

Reply:
> Maybe `HashType` instead? 

I thought about that one as well. Just seems to me that it doesn't convey the function/purpose of this enum.

> indicates we are dealking with a function, and not an enum

Maybe, although I think it's already defined as an enum, so that might be enough?

Reply:
yeah, maybe you're right. But definitely Method, not Function imo